File: src/test/java/be/cegeka/orderit/service/eventstore/api/LabelOnlyAnalysisRemovedEventTest.java
package be.cegeka.orderit.service.eventstore.api;

public class LabelOnlyAnalysisRemovedEventTest extends BaseInternalAnalysisEventTest {

    @Override
    protected AbstractInternalAnalysisEvent item() {
        final InternalAnalysisAggregateId aggregateIdentifier = new InternalAnalysisAggregateId("safdghjk");
        return new LabelOnlyAnalysisRemovedEvent(aggregateIdentifier, AnalysisNumber.from(45), "user");
    }
}

File: src/test/kotlin/be/cegeka/orderit/service/eventstore/api/LabelOnlyAnalysisRemovedEventTest.kt
package be.cegeka.orderit.service.eventstore.api

import be.cegeka.orderit.service.eventstore.api.AnalysisNumber.Companion.from

class LabelOnlyAnalysisRemovedEventTest : BaseInternalAnalysisEventTest() {

    override fun item(): AbstractInternalAnalysisEvent {
        val aggregateIdentifier = InternalAnalysisAggregateId("safdghjk")
        return LabelOnlyAnalysisRemovedEvent(aggregateIdentifier, from(45), "user")
    }
}